introduction/quickstart.md

```diff
@@ -0,0 +1,284 @@
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+## Quickstart with SQL using the GlareDB CLI
+
+1. Install and start the GlareDB binary by running the following in a shell:
+
+   ```shell
+   curl https://glaredb.com/install.sh | sh
+
+   ./glaredb
+   ```
+
+2. Start writing SQL! You can try out the below to query a Parquet file hosted
+   on GitHub over HTTP. This file contains NYC real estate transactions from
+   January 2022.
+
+   ```sql
+   SELECT *
+   FROM read_parquet('https://github.com/GlareDB/tutorial_data/raw/main/nyc_sales/sale_year=2022/sale_month=1/f55363e2587849bcb25c057be706c69d-0.parquet');
+   ```
+
+3. Download a file and query it locally. [Here is a file with the next month
+   of transaction data.] After downloading it, query it using its relative or
+   absolute path.
+
+   ```sql
+   -- Relative path to the same directory from which you started the GlareDB CLI
+   SELECT * FROM read_parquet('./f55363e2587849bcb25c057be706c69d-0.parquet');
+
+   -- Absolute path to the file
+   SELECT * FROM read_parquet('/Users/username/Downloads/f55363e2587849bcb25c057be706c69d-0.parquet');
+
+   ```
+
+4. Query a remote Postgres database using the `read_postgres()` function:
+
+   ```sql
+   SELECT * FROM read_postgres(
+   'postgresql://demo:demo@pg.demo.glaredb.com:5432/postgres', --connection
+   'public', --schema name
+   'customer' --table name
+   );
+   ```
+
+5. Join between the local Parquet file and the remote Postgres database to
+   get a count of number of sales per NYC borough, and include the names of the
+   NYC borough where the sales took place:
+
+   ```sql
+   SELECT
+       COUNT(sales.sale_date),
+       lookup.borough_name
+   FROM
+   read_parquet('https://github.com/GlareDB/tutorial_data/raw/main/nyc_sales/sale_year=2022/sale_month=1/f55363e2587849bcb25c057be706c69d-0.parquet') sales
+   JOIN
+       read_postgres(
+           'postgresql://demo:demo@pg.demo.glaredb.com:5432/postgres',
+           'public',
+           'borough_lookup'
+       ) lookup
+   ON sales.borough = lookup.borough_id
+   GROUP BY lookup.borough_name;
+   ```
+
+6. Save your data for easy access later using the previous query and `COPY TO`:
+
+   ```sql
+   COPY (
+       SELECT
+           COUNT(sales.sale_date),
+           lookup.borough_name
+       FROM
+           read_parquet('https://github.com/GlareDB/tutorial_data/raw/main/nyc_sales/sale_year=2022/sale_month=1/f55363e2587849bcb25c057be706c69d-0.parquet') sales
+       JOIN
+           read_postgres(
+               'postgresql://demo:demo@pg.demo.glaredb.com:5432/postgres',
+               'public',
+               'borough_lookup'
+           ) lookup
+       ON sales.borough = lookup.borough_id
+       GROUP BY lookup.borough_name
+   ) TO './my_joined_data.parquet';
+
+   SELECT *
+   FROM './my_joined_data.parquet';
+   ```
+
+7. If you're using [GlareDB Cloud], create new tables that will
+   persist in Cloud storage using:
```

Comment:
If you're not using glaredb cloud, what happens?

Reply:
When you close the connection the table goes away.

Reply:
at least for the CLI if you specify a local data directory, then data is persisted there. This is something we (could?) look into for the cloud. 

It would also maybe be a good thing to open a ticket to describe "create table" as something that should require some kind of specification (memory, local, cloud, external,) and also (in theory) some kind of format specification (delta, iceberg, flat-file, etc.)... 

We can and should have defaults (to some extent (particularly around format) but we should avoid situations where "this query executes differently if you have a cloud connection or not.